x-pack/plugins/apm/public/components/fleet_integration/apm_policy_form/settings_form/form_row_setting.tsx

```diff
@@ -106,6 +112,43 @@ export function FormRowSetting({ row, value, onChange }: Props) {
         />
       );
     }
+    case 'yaml': {
+      return (
+        <FixedHeightDiv>
+          <CodeEditor
+            languageId="yaml"
+            width="100%"
+            height="300px"
+            value={value}
+            onChange={(val) => {
+              onChange(row.key, val);
+            }}
+            options={{
+              minimap: {
```

Comment:
Nitpicking: it seems we can get rid off some options as they have been already defined as default values
for example, [code editor](https://github.com/elastic/kibana/blob/main/src/plugins/kibana_react/public/code_editor/code_editor.tsx#L413) and [glyphMargin](https://microsoft.github.io/monaco-editor/api/interfaces/monaco.editor.IEditorConstructionOptions.html#glyphMargin)

Reply:
I removed a couple of options